From 334dde1b91c3e4e255111f16598f48585c39cf0e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Fri, 18 Apr 2014 13:53:01 +0200
Subject: [PATCH 1/5] build with tvdaemon

---
 lib/libdvbv5/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/libdvbv5/Makefile.am b/lib/libdvbv5/Makefile.am
index f126dc5..2f67b20 100644
--- a/lib/libdvbv5/Makefile.am
+++ b/lib/libdvbv5/Makefile.am
@@ -105,10 +105,12 @@ libdvbv5_la_SOURCES = \
 	descriptors/desc_ca.c			\
 	descriptors/desc_ca_identifier.c
 
-libdvbv5_la_CPPFLAGS = -I../.. $(ENFORCE_LIBDVBV5_STATIC)
+libdvbv5_la_CPPFLAGS = -I../.. $(ENFORCE_LIBDVBV5_STATIC) -std=c99
 libdvbv5_la_LDFLAGS = $(LIBDVBV5_VERSION) $(ENFORCE_LIBDVBV5_STATIC) -lm
 libdvbv5_la_LIBADD = $(LTLIBICONV)
 
+AUTOMAKE_OPTIONS = subdir-objects
+
 EXTRA_DIST = README gen_dvb_structs.pl
 
 sync-with-kernel:
-- 
2.1.0

