From 467a01fe9347eb4023164a2b0b49bbffc42ca5b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sat, 5 Oct 2013 14:09:47 +0200
Subject: [PATCH 02/12] v4l: lvimrc

---
 .lvimrc | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 .lvimrc

diff --git a/.lvimrc b/.lvimrc
new file mode 100644
index 0000000..b8658a7
--- /dev/null
+++ b/.lvimrc
@@ -0,0 +1,4 @@
+set noexpandtab
+set shiftwidth=8
+set softtabstop=8
+
-- 
1.8.3.2

