From 8fef8b0e968050f0dc40ee0e1d398fc6b9aee76a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Mon, 4 Jun 2012 21:50:54 +0200
Subject: [PATCH 02/15] Fix dvb-file USER CMD
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: André Roth <neolynx@gmail.com>
---
 lib/libdvbv5/dvb-file.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/libdvbv5/dvb-file.c b/lib/libdvbv5/dvb-file.c
index 6431bbd..2c8905e 100644
--- a/lib/libdvbv5/dvb-file.c
+++ b/lib/libdvbv5/dvb-file.c
@@ -387,7 +387,7 @@ static int fill_entry(struct dvb_entry *entry, char *key, char *value)
 			break;
 	}
 	if (i < ARRAY_SIZE(dvb_v5_name)) {
-		const char * const *attr_name = dvb_v5_attr_names[i];
+		const char * const *attr_name = dvb_attr_names(i);
 		n_prop = entry->n_props;
 		entry->props[n_prop].cmd = i;
 		if (!attr_name || !*attr_name)
@@ -412,7 +412,7 @@ static int fill_entry(struct dvb_entry *entry, char *key, char *value)
 			break;
 	}
 	if (i < ARRAY_SIZE(dvb_user_name)) {
-		const char * const *attr_name = dvb_user_attr_names[i];
+		const char * const *attr_name = dvb_attr_names(i);
 		n_prop = entry->n_props;
 		entry->props[n_prop].cmd = i + DTV_USER_COMMAND_START;
 		if (!attr_name || !*attr_name)
-- 
1.7.9.5

