From ce5c7935cfb4fa72820cbcd581e6db6dd799f7d2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Thu, 3 Jan 2013 01:02:46 +0100
Subject: [PATCH] allow anonymous unions with gcc 4.5.5

---
 lib/libv4l-mplane/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/libv4l-mplane/Makefile.am b/lib/libv4l-mplane/Makefile.am
index 046f49f..c11b88a 100644
--- a/lib/libv4l-mplane/Makefile.am
+++ b/lib/libv4l-mplane/Makefile.am
@@ -3,5 +3,5 @@ libv4l2plugin_LTLIBRARIES = libv4l-mplane.la
 endif
 
 libv4l_mplane_la_SOURCES = libv4l-mplane.c
-libv4l_mplane_la_CPPFLAGS = -fvisibility=hidden -std=c99
+libv4l_mplane_la_CPPFLAGS = -fvisibility=hidden -std=gnu99
 libv4l_mplane_la_LDFLAGS = -avoid-version -module -shared -export-dynamic -lpthread
-- 
1.7.2.5

