From 653a038c62a1065a60e348f3aae240b24f6a596c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sun, 3 Aug 2014 14:48:35 +0200
Subject: [PATCH 7/7] libdvbv5: prevent compile warning

---
 lib/libdvbv5/dvb-sat.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/libdvbv5/dvb-sat.c b/lib/libdvbv5/dvb-sat.c
index 795524a..1ab18f9 100644
--- a/lib/libdvbv5/dvb-sat.c
+++ b/lib/libdvbv5/dvb-sat.c
@@ -18,6 +18,8 @@
  *
  */
 
+// for usleep
+#define _BSD_SOURCE
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
-- 
1.9.1

