From 8e41b73146669bae4af18e4c98b5c7f94514f0f0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Fri, 8 Jun 2012 00:33:13 +0200
Subject: [PATCH 07/15] vimrc

---
 .lvimrc |    8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 .lvimrc

diff --git a/.lvimrc b/.lvimrc
new file mode 100644
index 0000000..0da05a7
--- /dev/null
+++ b/.lvimrc
@@ -0,0 +1,8 @@
+set path+=lib/include
+
+set noexpandtab
+set tabstop=8
+set nosmarttab
+set shiftwidth=8
+set softtabstop=8
+
-- 
1.7.9.5

