From 1db26ec19ea867c05d0aff25af189699a2b7c289 Mon Sep 17 00:00:00 2001
From: Lars Schmohl <lars.schmohl@gmail.com>
Date: Fri, 8 Jun 2012 16:23:27 +0200
Subject: [PATCH 08/15] continue parsing the next descriptors

---
 lib/libdvbv5/descriptors.c |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/lib/libdvbv5/descriptors.c b/lib/libdvbv5/descriptors.c
index c333c7b..dd5992b 100644
--- a/lib/libdvbv5/descriptors.c
+++ b/lib/libdvbv5/descriptors.c
@@ -60,18 +60,24 @@ static char *table[] = {
 ssize_t dvb_parse_descriptor(const uint8_t *buf, uint8_t *dest, uint16_t section_length, struct dvb_desc **head_desc)
 {
 	ssize_t length = 0;
+	struct dvb_desc *last = NULL;
 	while (length < section_length) {
 		uint8_t type = buf[0];
+		uint8_t len  = buf[1];
 		if (dvb_descriptors[type].init) {
 			ssize_t len = dvb_descriptors[type].init( &buf, (struct dvb_desc *) dest );
 			if(!*head_desc)
 				*head_desc = (struct dvb_desc *) dest;
+			if (last)
+				last->next = (struct dvb_desc *) dest;
+			last = (struct dvb_desc *) dest;
 			dest += len;
 			length += len;
 
 		} else {
 			printf( "parsing of descriptor %s (%d) is not implemented yet\n", dvb_descriptors[type].name, type );
-			break;
+			buf += 2 + len;     /* standard descriptor header plus descriptor length */
+			length += 2 + len;  /* standard descriptor header plus descriptor length */
 		}
 	}
 	return length;
-- 
1.7.9.5

