From 1053e6e0b715361391c1fc7c25d87d22a857aae6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Mon, 30 Dec 2013 13:48:34 +0100
Subject: [PATCH 10/38] libdvbv5: README updated for shared libdvbv5
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: André Roth <neolynx@gmail.com>
---
 README        |  6 +++---
 README.libv4l | 12 ++++++++++++
 2 files changed, 15 insertions(+), 3 deletions(-)

diff --git a/README b/README
index c4f6c0c..a9f8089 100644
--- a/README
+++ b/README
@@ -3,11 +3,11 @@ v4l-utils
 
 Linux V4L2 and DVB API utilities and v4l libraries (libv4l).
 You can always find the latest development v4l-utils in the git repo:
-http://git.linuxtv.org/v4l-utils.git 
+http://git.linuxtv.org/v4l-utils.git
 
 
-v4l libraries (libv4l)
-----------------------
+v4l libraries (libv4l, libdvbv5)
+--------------------------------
 
 See README.libv4l for more information on libv4l, libv4l is released
 under the GNU Lesser General Public License.
diff --git a/README.libv4l b/README.libv4l
index ffe6366..d101c2c 100644
--- a/README.libv4l
+++ b/README.libv4l
@@ -59,6 +59,18 @@ hardware can _really_ do it should use ENUM_FMT, not randomly try a bunch of
 S_FMT's). For more details on the v4l2_ functions see libv4l2.h .
 
 
+libdvbv5
+--------
+
+This library provides the DVBv5 API to userspace programs. It can be used to
+open DVB adapters, tune transponders and read PES and other data streams.
+There are as well several parsers for DVB, ATSC, ISBT formats.
+
+The API is currently EXPERIMENTAL and likely to change.
+Run configure with --enable-libdvbv5 in order to build a shared lib and
+install the header files.
+
+
 wrappers
 --------
 
-- 
1.8.3.2

