From ae530ea7fdee4c8bc694d03ce2e4704fce852686 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Thu, 13 Jun 2013 17:33:20 +0200
Subject: [PATCH 2/2] cleanup coding style

---
 lib/libdvbv5/dvb-demux.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/libdvbv5/dvb-demux.c b/lib/libdvbv5/dvb-demux.c
index 2029204..3914a0a 100644
--- a/lib/libdvbv5/dvb-demux.c
+++ b/lib/libdvbv5/dvb-demux.c
@@ -51,13 +51,13 @@ int dvb_dmx_open(int adapter, int demux)
 
 void dvb_dmx_close(int dmx_fd)
 {
-  (void) ioctl( dmx_fd, DMX_STOP);
+  (void) ioctl(dmx_fd, DMX_STOP);
   close( dmx_fd);
 }
 
 void dvb_dmx_stop(int dmx_fd)
 {
-  (void) ioctl( dmx_fd, DMX_STOP);
+  (void) ioctl(dmx_fd, DMX_STOP);
 }
 
 int dvb_set_pesfilter(int dmxfd, int pid, dmx_pes_type_t type, dmx_output_t output, int buffersize)
-- 
1.7.10.4

