From d3a3380a72e8e4382be90e57e89fb238008b9c0e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sun, 29 Dec 2013 14:34:00 +0100
Subject: [PATCH 15/19] libdvbv5: cleanup dvb_nit_transport_foreach macro
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: André Roth <neolynx@gmail.com>
---
 lib/include/descriptors/nit.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/include/descriptors/nit.h b/lib/include/descriptors/nit.h
index d71a762..4f19c5e 100644
--- a/lib/include/descriptors/nit.h
+++ b/lib/include/descriptors/nit.h
@@ -76,8 +76,8 @@ typedef void nit_tran_handler_callback_t(struct dvb_table_nit *nit,
 					 struct dvb_desc *desc,
 					 void *priv);
 
-			       #define dvb_nit_transport_foreach( tran, nit ) \
-  for (struct dvb_table_nit_transport *tran = nit->transport; tran; tran = tran->next) \
+#define dvb_nit_transport_foreach( _tran, _nit ) \
+  for (struct dvb_table_nit_transport *_tran = _nit->transport; _tran; _tran = _tran->next) \
 
 struct dvb_v5_fe_parms;
 
-- 
1.8.3.2

