From 9cd596ee36c3e2f0bae310c4cbe8ff15840d6e7a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Wed, 8 Jan 2014 20:40:47 +0100
Subject: [PATCH 25/37] dvn scan

---
 lib/libdvbv5/dvb-scan.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/lib/libdvbv5/dvb-scan.c b/lib/libdvbv5/dvb-scan.c
index ced358c..ff441d1 100644
--- a/lib/libdvbv5/dvb-scan.c
+++ b/lib/libdvbv5/dvb-scan.c
@@ -191,6 +191,7 @@ int dvb_read_section_with_id(struct dvb_v5_fe_parms *parms, int dmx_fd,
 		}
 
 		h = (struct dvb_table_header *)buf;
+		// FIXME: atsc_table_header_init
 		dvb_table_header_init(h);
 
 		if (start_id == h->id && start_section == h->section_id) {
@@ -205,6 +206,8 @@ int dvb_read_section_with_id(struct dvb_v5_fe_parms *parms, int dmx_fd,
 			continue;
 		}
 
+		dvb_logdbg( "dvb_read_section: id %d, section %d/%d, current/next: %d", h->id, h->section_id, h->last_section, h->current_next );
+
 		/* handle the sections */
 		if (first_section == -1)
 			first_section = h->section_id;
@@ -242,7 +245,9 @@ int dvb_read_section_with_id(struct dvb_v5_fe_parms *parms, int dmx_fd,
 			break;
 		}
 	}
-	free(buf);
+
+	if (buf)
+		free(buf);
 
 	dvb_dmx_stop(dmx_fd);
 
-- 
1.8.3.2

