From cf5fc9e11774adebd781bff2d16b5c152462cc1f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Tue, 21 Jan 2014 17:57:10 +0100
Subject: [PATCH 27/41] fix desc printing

---
 lib/libdvbv5/descriptors.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/libdvbv5/descriptors.c b/lib/libdvbv5/descriptors.c
index 23bf1bd..78a8708 100644
--- a/lib/libdvbv5/descriptors.c
+++ b/lib/libdvbv5/descriptors.c
@@ -76,7 +76,6 @@ static void dvb_desc_default_print(struct dvb_v5_fe_parms *parms, const struct d
 {
 	if (!parms)
 		parms = dvb_fe_dummy();
-	dvb_log("|                   %s (0x%02x)", dvb_descriptors[desc->type].name, desc->type);
 	hexdump(parms, "|           ", desc->data, desc->length);
 }
 
-- 
1.8.3.2

