From bce665d76fb51d1b760a4e1a3b865ff73b790653 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sun, 9 Feb 2014 15:09:18 +0100
Subject: [PATCH 35/37] fix libdvbv5: remove unneeded dvb_fe_sec_tone

---
 lib/libdvbv5/dvb-sat.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/libdvbv5/dvb-sat.c b/lib/libdvbv5/dvb-sat.c
index b087a36..54df271 100644
--- a/lib/libdvbv5/dvb-sat.c
+++ b/lib/libdvbv5/dvb-sat.c
@@ -277,7 +277,7 @@ static int dvbsat_diseqc_set_input(struct dvb_v5_fe_parms *parms, uint16_t t)
 	int high_band = parms->high_band;
 	int sat_number = parms->sat_number;
 	int vol_high = 0;
-	int tone_on = 0;
+	/*int tone_on = 0;*/
 	int mini_b = 0;
 	struct diseqc_cmd cmd;
 
@@ -293,7 +293,7 @@ static int dvbsat_diseqc_set_input(struct dvb_v5_fe_parms *parms, uint16_t t)
 		/* Adjust voltage/tone accordingly */
 		if (parms->sat_number < 2) {
 			vol_high = pol_v ? 0 : 1;
-			tone_on = high_band;
+			/*tone_on = high_band;*/
 			mini_b = parms->sat_number & 1;
 		}
 	}
@@ -328,7 +328,7 @@ static int dvbsat_diseqc_set_input(struct dvb_v5_fe_parms *parms, uint16_t t)
 		usleep(15 * 1000);
 	}
 
-	rc = dvb_fe_sec_tone(parms, tone_on ? SEC_TONE_ON : SEC_TONE_OFF);
+	// not needed (tvh)! rc = dvb_fe_sec_tone(parms, tone_on ? SEC_TONE_ON : SEC_TONE_OFF);
 
 	return rc;
 }
-- 
1.8.3.2

