From eae3cfccef6da4a19206ca0084b2f89c08216024 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Mon, 10 Feb 2014 19:38:29 +0100
Subject: [PATCH 37/41] free desc memory on error

---
 lib/libdvbv5/descriptors.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/libdvbv5/descriptors.c b/lib/libdvbv5/descriptors.c
index c2239bb..8e24cc6 100644
--- a/lib/libdvbv5/descriptors.c
+++ b/lib/libdvbv5/descriptors.c
@@ -187,11 +187,12 @@ int dvb_parse_descriptors(struct dvb_v5_fe_parms *parms, const uint8_t *buf,
 
 		current = malloc(size);
 		if (!current) {
-			dvb_perror("Out of memory");
+			dvb_logerr("%s: out of memory", __func__);
 			return -3;
 		}
 		dvb_desc_init(desc_type, desc_len, current); /* initialize the standard header */
 		if (init(parms, ptr, current) != 0) {
+			free(current);
 			return -4;
 		}
 		if (!*head_desc)
-- 
1.8.3.2

