From 8664ee6ce8534fd3f1bf682b09d8ba0432e28216 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sat, 22 Feb 2014 18:58:03 +0100
Subject: [PATCH 40/40] fix conflict

---
 lib/libdvbv5/descriptors.c |   12 ------------
 1 file changed, 12 deletions(-)

diff --git a/lib/libdvbv5/descriptors.c b/lib/libdvbv5/descriptors.c
index d7060d0..4857936 100644
--- a/lib/libdvbv5/descriptors.c
+++ b/lib/libdvbv5/descriptors.c
@@ -126,17 +126,11 @@ int dvb_parse_descriptors(struct dvb_v5_fe_parms *parms, const uint8_t *buf,
 	struct dvb_desc *current = NULL;
 	struct dvb_desc *last = NULL;
 
-<<<<<<< HEAD
-	while (ptr + 2 < buf + section_length) {
-		unsigned desc_type = ptr[0];
-		int desc_len  = ptr[1];
-=======
 	*head_desc = NULL;
 
 	while (ptr + 2 <= endbuf ) {
 		uint8_t desc_type = ptr[0];
 		uint8_t desc_len  = ptr[1];
->>>>>>> libdvbv5: fix dvb_parse_descriptors and make dvb_desc_init private
 		size_t size;
 
 		if (desc_type == 0xff ) {
@@ -167,12 +161,6 @@ int dvb_parse_descriptors(struct dvb_v5_fe_parms *parms, const uint8_t *buf,
 			hexdump(parms, "content: ", ptr + 2, desc_len);
 		}
 
-		if (desc_len > section_length - 2) {
-			dvb_logwarn("descriptor type 0x%02x is too big",
-				   desc_type);
-			return;
-		}
-
 		dvb_desc_init_func init = dvb_descriptors[desc_type].init;
 		if (!init) {
 			init = dvb_desc_default_init;
-- 
1.7.10.4

