From 4d2e29e06c57c351d99bb4d991d724325226129b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Roth?= <neolynx@gmail.com>
Date: Sat, 11 Aug 2012 20:28:15 +0200
Subject: [PATCH 2/2] Do not remove default items

---
 src/CLI.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/CLI.py b/src/CLI.py
index 7df8b81..659bfd8 100644
--- a/src/CLI.py
+++ b/src/CLI.py
@@ -183,7 +183,7 @@ class CLI:
     for i in self._items:
       if i.Category in categories:
         i.Enabled = enabled
-      elif other_inverse:
+      elif other_inverse and i.Category != "default":
         i.Enabled = not enabled
 
   def GetItemByCLILine( self, line ):
-- 
1.7.2.5

